src/instructlab/model/train.py

```diff
@@ -279,9 +303,67 @@
 @click.option(
     "--legacy",
     is_flag=True,
-    default=False,
     help="if true, enables the legacy linux training codepath from release 0.17.0 and prior.",
 )
+@click.option(
+    "--strategy",
+    type=click.Choice(["lab-multiphase"], case_sensitive=False),
+    show_default=True,
+    help="If chosen, will run the selected training strategy instead of a single training run.",
+)
+@click.option(
+    "--phased-base-dir",
+    type=clickpath_setup(is_dir=True),
+    default=lambda: DEFAULTS.PHASED_DIR,
+    help="Base directory for organization of end-to-end intermediate outputs.",
+)
+@click.option(
+    "--phased-phase1-data",
+    type=clickpath_setup(is_dir=False),
+    help="Path to .jsonl file that will be used for the first phase of end-to-end training.",
+)
+@click.option(
```

Comment:
not going to block on any of this but we could combine all of these flags.

Any flag that has a phase number in it could be combined into a single flag like `--phased-num-epochs` or `--phased-samples-per-save` with the `multiple=True`. This allows multiple values to be given as the input so you could give `--phased-num-epochs=[7, 10]` or something

Reply:
Sure could do that. That'd be a good follow-up for neatness.